<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LaShawn" failures="1" tests="12" name="firefoxTest" time="104.336" errors="0" timestamp="2025-08-28T23:53:21 EDT">
  <testcase classname="testCases.TC01_LaunchApplication" name="testLaunchApplication" time="0.063"/>
  <testcase classname="testCases.TC03_AddToCart" name="testAddToCart" time="8.511"/>
  <testcase classname="testCases.TC05_AddToWishList" name="testAddToWishList" time="5.025"/>
  <testcase classname="testCases.TC06_AddAffiliate" name="testAddAffiliate" time="5.175"/>
  <testcase classname="testCases.TC02_Login" name="testLogin" time="2.386"/>
  <testcase classname="testCases.TC02_Login" name="testLogin" time="1.242"/>
  <testcase classname="testCases.TC02_Login" name="testLogin" time="1.327"/>
  <testcase classname="testCases.TC02_Login" name="testLogin" time="1.217"/>
  <testcase classname="testCases.TC02_Login" name="testLogin" time="0.972"/>
  <testcase classname="testCases.TC04_CompletePurchase" name="testCompletePurchase" time="6.489">
    <skipped/>
  </testcase> <!-- testCompletePurchase -->
  <testcase classname="testCases.TC04_CompletePurchase" name="testCompletePurchase" time="1.862">
    <skipped/>
  </testcase> <!-- testCompletePurchase -->
  <testcase classname="testCases.TC04_CompletePurchase" name="testCompletePurchase" time="1.874">
    <failure type="java.lang.AssertionError" message="Test failed due to unexpected exception.">
      <![CDATA[java.lang.AssertionError: Test failed due to unexpected exception.
at testCases.TC04_CompletePurchase.testCompletePurchase(TC04_CompletePurchase.java:81)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- testCompletePurchase -->
</testsuite> <!-- firefoxTest -->
